lua/obsidian/command.lua

```diff
@@ -6,6 +6,17 @@ local util = require "obsidian.util"
 
 local command = {}
 
+local opts = {
+  use_advanced_uri = nil,
+}
+
+--- Inject init/setup dependencies here (internal use)
+---
+---@param use_advanced_uri boolean|?
+command.setup = function(use_advanced_uri)
+  opts.use_advanced_uri = use_advanced_uri
+end
```

Comment:
I don't think you need all this because you can just access `client.opts.use_advanced_uri` directly.

Reply:
I had a feeling I was doing something not quite right. Give me a sec.